griptape/drivers/observability/dummy_observability_driver.py

```diff
@@ -0,0 +1,6 @@
+from griptape.drivers.observability.base_observability_driver import BaseObservabilityDriver
+
+
+class DummyObservabilityDriver(BaseObservabilityDriver):
+    def invoke_observable(self, func, instance, args, kwargs, decorator_args, decorator_kwargs):
+        return func(*args, **kwargs)
```

Comment:
Should raise a `DummyException` like the other Drivers.

Reply:
We don't actually need a `DummyObservabilityDriver`, that's just me abusing the conventions.

This was actually intended as a `NoOpObservabilityDriver`, for (1) use in unit tests and (2) illustrate a bare minimum example of subclassing `BaseObservabilityDriver`. If you don't think (2) is important, I can move this into a unit test utility.

Reply:
I forgot one other reason for this, it's the default implementation of the `@observable` decorator, which applies when not using the observability feature.

Since this is not just for illustration + unit testing, I'll leave the class where it is, but I'll rename it to `NoOpObservabilityDriver` as to not abuse conventions.

Leaving this comment unresolved in case you have follow up comments or questions.

Reply:
If it was just for unit-testing purposes, i'd say move to `tests.mocks`. If it's the default implementation can we just move the logic to the base class and get rid of this entirely?